api/src/main/java/org/openmrs/validator/ObsValidator.java

```diff
@@ -319,11 +309,47 @@ public ConceptReferenceRange getReferenceRange(Obs obs) {
 		}
 
 		if (validRanges.isEmpty()) {
-			return null;
+			ConceptReferenceRange defaultReferenceRange = getDefaultReferenceRange(conceptNumeric);
+			if (defaultReferenceRange != null) {
+				validRanges.add(defaultReferenceRange);
+			} else {
+				return null;
+			}
 		}
+		
 		return findStrictestReferenceRange(validRanges);
 	}
-	
+
+	/**
+	 * Loads the reference range from the ConceptNumeric if no reference ranges are associated with
+	 * this concept and person.
+	 * 
+	 * @param conceptNumeric A {@link ConceptNumeric} to extract the default values from
+	 * @return a {@link ConceptReferenceRange} containing the reference range from the concept
+	 */
+	private static ConceptReferenceRange getDefaultReferenceRange(ConceptNumeric conceptNumeric) {
+		if (conceptNumeric == null || (
+			conceptNumeric.getHiAbsolute() == null &&
+			conceptNumeric.getHiCritical() == null &&
+			conceptNumeric.getHiNormal() == null &&
+			conceptNumeric.getLowAbsolute() == null &&
+			conceptNumeric.getLowCritical() == null &&
+			conceptNumeric.getLowNormal() == null
+		)) {
+			return null;
+		}
+		
+		ConceptReferenceRange defaultReferenceRange = new ConceptReferenceRange();
+		defaultReferenceRange.setConceptNumeric(conceptNumeric);
+		defaultReferenceRange.setHiAbsolute(conceptNumeric.getHiAbsolute());
+		defaultReferenceRange.setHiCritical(conceptNumeric.getHiCritical());
+		defaultReferenceRange.setHiNormal(conceptNumeric.getHiNormal());
+		defaultReferenceRange.setLowAbsolute(conceptNumeric.getLowAbsolute());
+		defaultReferenceRange.setLowCritical(conceptNumeric.getLowCritical());
+		defaultReferenceRange.setLowNormal(conceptNumeric.getLowNormal());
+		return defaultReferenceRange;
+	}
+
 	/**
 	 * Finds the strictest {@link ConceptReferenceRange} from a list of valid ranges.
 	 * The strictest range is determined by having the highest lower bound and the lowest upper bound.
```

Comment:
Since this is changing the definition of strictest range, should't the above comment also change to reflect it?